tests/protocols/test_websocket.py

```diff
@@ -756,9 +756,7 @@ async def open_connection(url):
     )
     async with run_server(config):
         headers = await open_connection("ws://127.0.0.1:8000")
-        assert all(
-            x in headers.get_all("Server") for x in ["over-ridden", "another-value"]
-        )
+        assert headers.get_all("Server") == ["uvicorn", "over-ridden", "another-value"]
```

Comment:
Does this pass the test?

Reply:
Yes it does. I will merge this to the other branch

Reply:
thanks